### DISCUSSION TITLE
Timeouts on fakerphp/faker & google/apiclient-services

### DISCUSSION THREAD

**Discussion by warmwhisky** [composer/composer#11185] [Q&A / Support]
Is anyone having issues with composer update? I cannot update as fakerphp/faker & google/apiclient-services keep timing out. This started yesterday early morning. I left it over night and still did not update

I used composer update -vvv and could see redirects to https://codeload.github.com/googleapis/google-api-php-client-services/legacy.zip/670c3854fffc2f642efa86b083e2664fd55435e1 trying to download these manually seems to fail too

It does not seem to be project specific either as its the same on another project.

I have also tried on two different internet connections.

I am using `Composer version 2.4.1` with Laravel 9.9 and Docker

**Comment by stof** [comment]
If the manual download also fails, you should probably contact the GitHub support about it.

**Reply by warmwhisky** [reply]
I was not able to confirm if anyone else is having this issue or whether its something to do with my ISP or the ISPs in Swaziland. We have political unrest this week and our Gov tend to block sites, but I cannot confirm this.

I am guessing you have no issue with the manual download link?

**Reply by stof** [reply]
No indeed. The download link works for me.

**Reply by warmwhisky** [reply]
I'm at a loss here. This has been going on for 2 full work days. Here is a screenshot of waiting an hour with composer update 

![Screenshot from 2022-11-15 15-22-43](https://user-images.githubusercontent.com/579186/201930204-37f807da-f35c-4252-9524-72dec583bce1.png)

**Reply by warmwhisky** [reply]
Thanks for your time. I figured out last night by using internet from South Africa that it was my ISP. I still don't understand why it happened with mostly fakerphp/faker & google/apiclient-services.

This is solved now.

**Reply by stof** [reply]
Probably because those packages are bigger than others
